### PR TITLE
taha/LP-1699 - Fix - category access issue with apostrophe in course name

### PR DESCRIPTION
**Pull request by arbisofttaha**

### CODE REVIEW

File: src/categories/data.js [hunk 1]
```diff
@@ -7,13 +7,15 @@ var winston = require('winston');
...
 module.exports = function (Categories) {
-	Categories.getCategoryData = function (cid, callback) {
+	Categories.getCategoryData = function (cid, callback, isModificationBlocked) {
 		async.waterfall([
 			function (next) {
 				db.getObject('category:' + cid, next);
 			},
 			function (category, next) {
-				modifyCategory(category);
+				if (!isModificationBlocked) {
```
Comment (ahmed-faraz46) [inline src/categories/data.js:16]: Let's please document the reason, why do we need to place this check.

**Review by ahmed-faraz46 — CHANGES_REQUESTED** [review]

File: src/categories/data.js (hunk 1 above)
Comment (arbisofttaha) [reply src/categories/data.js:16]: Comment added for documentation purposes.

Comment (ahmed-faraz46) [reply src/categories/data.js:16]: @arbisofttaha Actually I wanted to document the exact edge case we tackled here.

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM :+1:

**Review by mabdullahz — APPROVED** [review]
LGTM :+1:

**Pull request merged**